src/components/VtkTwoView.vue

```diff
@@ -452,7 +505,7 @@ export default defineComponent({
         }
 
         // TODO listen to changes in point data
-        const range = imageData.getPointData().getScalars().getRange();
+        const range = autoRangeValues.value[autoRange.value];
```

Comment:
This TODO no longer applies to this line and can be removed.

Reply:
Thanks! Removed.